CHANGELOG.md

```diff
@@ -3,3 +3,18 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [0.1.0] - 2018-08-28
+### Added
+- initial commit https://github.com/ExpediaInc/apiary-ranger-docker/issues/1
+- Dockerfile to build common docker image for running ranger-admin & ranger-userssync
+- ranger-admin & ranger-usersync startup scripts
+    startup script also initializes database
```

Comment:
I'm not sure whether we need this level of detail in the CHANGELOG, up to you guys, we normally just have one liners for each change and then link through to the GH issue for more detail